src/Toolchain/ToolchainProvider.ts

```diff
@@ -27,15 +28,21 @@ export class ToolchainNode extends vscode.TreeItem {
       public readonly label: string,
       public readonly collapsibleState: vscode.TreeItemCollapsibleState,
       public readonly type: NodeType,
-      public readonly version?: string,
+      public readonly toolchain?: Toolchain,
   ) {
     super(label, collapsibleState);
 
     if (type === NodeType.backend) {
       this.iconPath = new vscode.ThemeIcon('bracket');
     } else if (type === NodeType.toolchain) {
+      if (toolchain === undefined) {
+        throw Error('Invalid ToolchainNode');
+      }
       this.iconPath = new vscode.ThemeIcon('circle-filled');
-      this.label = `${this.label} ${this.version}`;
+      this.description = toolchain.info.version ?.str();
```

Comment:
(This comment can be ignored when merging)
Is it intentional whitespace? :D

```suggestion
      this.description = toolchain.info.version?.str();
```

Reply:
@jyoungyun said that format checker did this... wow @.@